proto/osmosis/incentives/query.proto

```diff
@@ -124,6 +131,16 @@ message UpcomingGaugesResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+message UpcomingGaugesPerDenomRequest {
+  string denom = 1;
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
```

Comment:
We gotta be more careful with reviewing proto structs.

```suggestion
  string denom = 1;
  cosmos.base.query.v1beta1.PageRequest pagination = 2;
```

Reply:
Oop! Updated.

Reply:
Yeah your totally right. Is there a way we can lint for this more easily? I don't get why make proto-all even compiled with this as an issue.